packages/slice-machine/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "slice-machine-ui",
-  "version": "0.5.1-dev.1",
+  "version": "0.5.1-dev.14",
```

Comment:
you didn't change the versions of the core and Clients in the dependencies to match the new versions.